autoagents/eval/test_v3.py

```diff
@@ -0,0 +1,73 @@
+import asyncio
```

Comment:
Do we really need a separate test file for v3?

Reply:
Not really, at this point the file is just acting as documentation on how to use prompt v3. The eval MR has done a major refactoring to this file and should just assimilate the prompt v3 changes and remove the file.